frontend-react/src/components/Snitch.tsx

```diff
@@ -1,9 +1,11 @@
 import React, {CSSProperties, useEffect, useState} from 'react';
 import {Stomp, StompSubscription} from '@stomp/stompjs';
-import {DateTimeFormatter, LocalDateTime, Month} from '@js-joda/core';
+import {DateTimeFormatter, LocalDateTime} from '@js-joda/core';
 import {Card, Col, Container, Row, Table} from "react-bootstrap";
 import xmlFormatter from 'xml-formatter';
 
+import {parse} from 'content-type'
```

Comment:
Her kan du gjøre import alias:

`import {parse as parseContentType} from 'content-type';`

for at koden lengre nede skal være mer lesbar.